eos/models/wet.cc

```diff
@@ -579,6 +579,74 @@ namespace eos
         return result;
     }
 
+    std::array<std::tuple<UsedParameter, UsedParameter>, 20>
+    make_wet_parameters_classVhadronic_full(const Parameters & p, ParameterUser & u, const QuarkFlavor & q)
+    {
+        const auto base_prefix = "sb";
+        std::string prefix, quark_flavour_string;
+        switch (q)
+        {
+            case QuarkFlavor::charm:
+                quark_flavour_string = "c";
+                break;
+            case QuarkFlavor::up:
+                quark_flavour_string = "u";
+                break;
+            case QuarkFlavor::down:
+                quark_flavour_string = "d";
+                break;
+            case QuarkFlavor::strange:
+            case QuarkFlavor::bottom:
+            default:
+                throw InternalError("ClassVhadronic::full WCs only implemented for u,d,c");
+        }
```

Comment:
I'm not convinced having a `QuarkFlavor` argument and then doing a switch/case is much nicer than just passing in the variable quark prefix as a string, and checking it is either `cc`, `uu`, or `dd`